### PR TITLE
Add table oci_identity_tag_namespace. Closes #104

### LINKED ISSUE #104: Add table oci_identity_tag_namespace
**References**
It will help to query CIS > 3.2 Ensure default tags are used on resources.

 
**Reference link**
https://docs.oracle.com/en-us/iaas/Content/Tagging/Tasks/managingtagsandtagnamespaces.htm

### PR DESCRIPTION
**Pull request by bigdatasourav**
# Integration test logs
<details>
  <summary>Logs</summary>
  
```
SETUP: tests/oci_identity_tag_namespace []

PRETEST: tests/oci_identity_tag_namespace

TEST: tests/oci_identity_tag_namespace
Running terraform
oci_identity_tag_namespace.named_test_resource: Creating...
oci_identity_tag_namespace.named_test_resource: Creation complete after 1s [id=ocid1.tagnamespace.oc1..aaaaaaaaiszsnwji72pct5gclwskwwgjaogxl3ecata5hdxxes7cnqjzdona]

Apply complete! Resources: 1 added, 0 changed, 0 destroyed.

Outputs:

resource_id = "ocid1.tagnamespace.oc1..aaaaaaaaiszsnwji72pct5gclwskwwgjaogxl3ecata5hdxxes7cnqjzdona"
resource_name = "steampipetest5677"
tag_namespace_description = "Terraform testing resource"
tenancy_ocid = "ocid1.tenancy.oc1..aaaaaaaahnm7gleh5soecxzjetci3yjjnjqmfkr4po3hoz4p4h2q37cyljaq"

Running SQL query: test-get-query.sql

[
  {
    "description": "Terraform testing resource",
    "id": "ocid1.tagnamespace.oc1..aaaaaaaaiszsnwji72pct5gclwskwwgjaogxl3ecata5hdxxes7cnqjzdona",
    "lifecycle_state": "ACTIVE",
    "name": "steampipetest5677"
  }
]
✔ PASSED

Running SQL query: test-list-query.sql

[
  {
    "description": "Terraform testing resource",
    "id": "ocid1.tagnamespace.oc1..aaaaaaaaiszsnwji72pct5gclwskwwgjaogxl3ecata5hdxxes7cnqjzdona",
    "lifecycle_state": "ACTIVE",
    "name": "steampipetest5677"
  }
]
✔ PASSED

Running SQL query: test-not-found-query.sql

null
✔ PASSED

Running SQL query: test-turbot-query.sql

[
  {
    "tenant_id": "ocid1.tenancy.oc1..aaaaaaaahnm7gleh5soecxzjetci3yjjnjqmfkr4po3hoz4p4h2q37cyljaq",
    "title": "steampipetest5677"
  }
]
✔ PASSED

POSTTEST: tests/oci_identity_tag_namespace

TEARDOWN: tests/oci_identity_tag_namespace

SUMMARY:

1/1 passed.
```
</details>

# Example query results
<details>
  <summary>Results</summary>
  
```
select
  name,
  id,
  is_retired,
  lifecycle_state
from
  oci_identity_tag_namespace;
+-------------+--------------------------------------------------------------------------------------+------------+-----------------+
| name        | id                                                                                   | is_retired | lifecycle_state |
+-------------+--------------------------------------------------------------------------------------+------------+-----------------+
| test        | ocid1.tagnamespace.oc1..aaaaaaaa4663ci3w42ujavuxoakkikxokgzrwmyfdstxin3exnmdldg6hnca | false      | ACTIVE          |
| Oracle-Tags | ocid1.tagnamespace.oc1..aaaaaaaal44hh2jnn3wbuvo7bb73poa4qpdvp6zzunynqh76fcn4jgld2h4q | false      | ACTIVE          |
+-------------+--------------------------------------------------------------------------------------+------------+-----------------+

 select
  name,
  id,
  is_retired,
  lifecycle_state
from
  oci_identity_tag_namespace
where
  lifecycle_state = 'ACTIVE';
+-------------+--------------------------------------------------------------------------------------+------------+-----------------+
| name        | id                                                                                   | is_retired | lifecycle_state |
+-------------+--------------------------------------------------------------------------------------+------------+-----------------+
| test        | ocid1.tagnamespace.oc1..aaaaaaaa4663ci3w42ujavuxoakkikxokgzrwmyfdstxin3exnmdldg6hnca | false      | ACTIVE          |
| Oracle-Tags | ocid1.tagnamespace.oc1..aaaaaaaal44hh2jnn3wbuvo7bb73poa4qpdvp6zzunynqh76fcn4jgld2h4q | false      | ACTIVE          |
+-------------+--------------------------------------------------------------------------------------+------------+-----------------+


select
  name,
  id,
  is_retired,
  lifecycle_state
from
  oci_identity_tag_namespace
where
  is_retired;
+------+--------------------------------------------------------------------------------------+------------+-----------------+
| name | id                                                                                   | is_retired | lifecycle_state |
+------+--------------------------------------------------------------------------------------+------------+-----------------+
| test | ocid1.tagnamespace.oc1..aaaaaaaa4663ci3w42ujavuxoakkikxokgzrwmyfdstxin3exnmdldg6hnca | true       | INACTIVE        |
+------+--------------------------------------------------------------------------------------+------------+-----------------+
```
</details>

### CODE REVIEW

File: oci/table_oci_identity_tag_namespace.go [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+
+			// Other columns
+			{
+				Name:        "description",
+				Description: "The description you assign to the tag namespace.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "time_created",
+				Description: "Date and time the tagNamespace was created",
```
Comment (khushboo9024) [inline oci/table_oci_identity_tag_namespace.go:60]:
```suggestion
				Description: "Date and time the tagNamespace was created.",
```
`time_created` is important column, it should be above `description`.

**Review by khushboo9024 — CHANGES_REQUESTED** [review]
Integration test logs attached  is of `azure_security_center` , please update the logs.

**Review by khushboo9024 — APPROVED** [review]
LGTM

**Review by rajmohanty17 — APPROVED** [review]
LGTM

**Review by cbruno10 — APPROVED** [review]

**Pull request merged**